mne/preprocessing/ecg.py

```diff
@@ -177,8 +164,12 @@ def find_ecg_events(raw, event_id=999, ch_name=None, tstart=0.0,
         The events corresponding to the peaks of the R waves.
     ch_ecg : string
         Name of channel used.
-    average_pulse : float
-        Estimated average pulse.
+    average_pulse : float | np.nan
+        The estimated average pulse. If no ECG events could be found, this will
+        be ``nan``.
```

Comment:
is this accurate?  Do we ever return `np.nan`? It looks like the part of the PR that changed that in the code has been removed.

Reply:
Good catch. I have corrected this.